app/helpers/requests/application_helper.rb

```diff
@@ -71,6 +71,8 @@ def hidden_service_options(requestable)
       # temporary #348
       elsif requestable.services.include? 'on_shelf'
         request_input('on_shelf')
+      else
+        request_input(requestable.services.first)
```

Comment:
I added this line because the commented out line 65 wasn't returning data as far as I could tell and only the "recap" options were getting the request type attribute set. Just trying to see if I could get the form to work for all request types. This definitely needs to be done differently but this method should always return a type attribute.